tools/tributors_to_citation.py

```diff
@@ -0,0 +1,54 @@
+import json
+import ruamel.yaml
+
+yaml = ruamel.yaml.YAML()
+
+citation_file = "/home/remi/github/BIDS-specification/CITATION.cff"
```

Comment:
TOO: ship along?

Reply:
Wanted to make sure there was some interest for the general direction of this PR before working on some final version of the script.

Don't say "I am pro open source but say I am a code exhibitionist."